src/System.Data.SqlClient/src/System/Data/SqlClient/SqlMetadataFactory.cs

```diff
@@ -1,289 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I merged my fork with upstream and that was edited and since I did not plan to merge the PR since I just wanted to trigger an OSX run I left the file in the PR